### PR TITLE
React and Preact recipes

### PR DESCRIPTION
**Pull request by jamesmcallister**
Added React and Preact to recipes to the docs.

### CODE REVIEW

File: src/docs/recipes.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+```
+
+Add Start script to `package.json`
+
+```js
+// package.json
+"scripts": {
+  "start": "parcel index.html"
+  }
```
Comment (thejameskyle) [inline src/docs/recipes.md]: indentation

File: src/docs/recipes.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
```
Comment (thejameskyle) [inline src/docs/recipes.md]: ^

File: src/layout/page.html [hunk 3]
```diff
@@ -29,19 +31,42 @@
...
+          <a href="asset_types.html">📝 Asset Types</a>
+        </li>
+        <li>
+          <a href="packagers.html">📦 Packagers</a>
+        </li>
+        <li>
+          <a href="plugins.html">🔌 Plugins</a>
+        </li>
+        <li>
+          <a href="recipes.html">🍰 Recipes</a>
```
Comment (jakoblind) [inline src/layout/page.html]: I don't think the Recipes section should be under "Advanced". It contains getting started recipes.

File: src/docs/recipes.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+# 🍰 Recipes
+
+## React
+
+First need to install the dependencies for React.
+
+[Blog Post](http://blog.jakoblind.no/react-parcel/)
+
+```
+npm install --save react react-dom npm install --save-dev parcel-bundler
```
Comment (jakoblind) [inline src/docs/recipes.md]: each npm command should be on it's own row.

**Comment by jakoblind** [comment]
Also, should we have a link to these getting started recipes on the getting started page? (https://parceljs.org/getting_started.html)

**Pull request merged**